msprime/ancestry.py

```diff
@@ -1072,6 +1072,9 @@ def sim_ancestry(
         greater than or equal to 1. For continuous genomes the tract lengths are
         exponentially distributed with mean ``gene_conversion_tract_length``,
         which must be larger than 0.
+    :param population_size: The effective (diploid) population size for the reference
```

Comment:
This isn't quite right I think. How about 

> The size of the default single population :class:`.Demography. If not specified, defaults to 1. Cannot be specified along with the ``demography`` parameter. See the :ref:`sec_demography` section for more details on demographic models and population sizes.

Reply:
Also we should refer to ``sec_ancestry_population_size`` for examples.

Reply:
Ok, so how's this:
> :param population_size: The size of the default single population :class:`.Demography`. If not specified, defaults to 1. Cannot be specified along with the ``demography`` parameter. See the :ref:`sec_demography`section for more details on demographic models and population sizes and the :ref:`sec_ancestry_population_size` section for usage examples.

Should it read `the default single population _of_  :class:`.Demography``?

Reply:
yep, LGTM. I think "single population Demography" is ok, "single population of Demography" wouldn't make sense? (The :class: things are markup)